### DISCUSSION TITLE
Bind Rive c++ to have it along with Lottie (skottie)

### DISCUSSION THREAD

**Discussion by taublast** [mono/SkiaSharp#2455] [Ideas]
Rive is coming along, and would be nice to adopt this tech on time.
https://rive.app/
https://github.com/rive-app

point of interest: https://github.com/rive-app/rive-cpp

This is a suggestion, if anyone would have the time to do it would be huge.

**Comment by wieslawsoltes** [comment]
https://github.com/rive-app/rive-sharp
https://www.nuget.org/packages/Rive.RiveSharp/

**Reply by taublast** [reply]
wow thanks totally missed that! looks like this week ill have something to play with =)

**Reply by michael-hawker** [reply]
We have a control wrapper around it for WinUI 2/3 and Uno Platform at https://aka.ms/toolkit/labs/windows as well.
